CONFIGURATION-GUIDE.md

```diff
@@ -299,6 +300,30 @@ Each workload receives a small set of monitoring Alarms per workload as well as
 * [enable_workload_warning_alarm](VARIABLES.md#input_enable_workload_warning_alarm) - This enables all workload warning alarms on deployment of LZ. (Boolean) Default `false`
 * [enable_workload_critical_alarm](VARIABLES.md#input_enable_workload_critical_alarm) - This enables all workload critical alarms on deployment of LZ. (Boolean) Default `false`
 
+## Multi-R[compartments.tf](compartments.tf)egion
+The Landing Zone can be deployed in a non-home region as long as there has already been a successful Landing Zone deployment in the home region and the non-home region is a designated paired region with the home region.
+
+This deployment can be controlled using the [home_region_deployment](VARIABLES.md#input_home_region_deployment) variable. This variable is set to `true` by default, which deploys all standard Landing Zone compartments and resources in the home region.
+
+If the [home_region_deployment](VARIABLES.md#input_home_region_deployment) variable is set to `false`, then the Landing Zone is configured for a non-home region deployment. This creates all the standard Landing Zone resources previously deployed in the home region, **except** identity resources such as compartments, policies, and domains. This is because identity resources can only be created in the home region.
+
+To deploy to a non-home region using the Terraform CLI, follow steps 1-3 in the [Implementation Guide](IMPLEMENTATION.md). Next, create a separate folder outside the Landing Zone root directory named home_region, and copy the terraform.tfstate, terraform.tfstate.backup, and terraform.tfvars files from the previous home region deployment into that folder. If you wish to make any future stack changes in the home region, you can move the contents of the home_region folder back into the Landing Zone root directory.
```

Comment:
To my understanding, we would like to have a folder without the terraform state of the home-region deployment. How about just initiating a new stack/folder so the user will not remove or add back the terraform state file since it's a little bit confusing to manage the state file? One stack/folder for home-region deployment, another stack/folder for non-home region deployment, that may be easier and clearer to manage the resources.

Reply:
Good suggestion, that could definitely be confusing. I'll edit that.